server/cmd/server/wallet.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY for erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th-----
```

Comment:
do not put wallet.pem in here. only public keys. The private key has to be a setup requirement when starting the node, and has to come from other place.

Reply:
Deleted it